include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -479,6 +479,11 @@ struct IoBinding : public Base<OrtIoBinding> {
   void ClearBoundOutputs();
 };
 
+struct ArenaCfg : Base<OrtArenaCfg> {
+  explicit ArenaCfg(std::nullptr_t) {}
+  ArenaCfg(size_t max_mem, int arena_extend_strategy, int initial_chunk_size_bytes, int max_dead_bytes_per_chunk);
```

Comment:
>ArenaCfg [](start = 2, length = 8)

perhaps a factory method would be a good idea?

Reply:
But why though ? Why move from an established pattern ? Can you please elaborate on why it will be a good idea for this ?